cdap-client-tests/src/main/java/co/cask/cdap/client/app/AppReturnsArgs.java

```diff
@@ -49,7 +50,8 @@ public void configure() {
     @Path(ENDPOINT)
     @GET
     public void handler(HttpServiceRequest request, HttpServiceResponder responder) {
-      responder.sendJson(200, getContext().getRuntimeArguments());
+      TreeMap<String, String> values = new TreeMap<>(getContext().getRuntimeArguments());
```

Comment:
what is the reason for failure ? because right now in the test case the expected value is of type hashmap.